core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -172,7 +172,6 @@ public WalletsSetup(RegTestHost regTestHost,
 
         btcWalletFileName = "bisq_" + config.baseCurrencyNetwork.getCurrencyCode() + ".wallet";
         params = Config.baseCurrencyNetworkParameters();
-        PeerGroup.setIgnoreHttpSeeds(true);
```

Comment:
Are httpSeed still used by default in BitcoinJ? If so I would prefer to still make sure to not use http seeds as they have some privacy issues.

Reply:
http seeds are not used by default in bitcoinj and are not used in bisq as is.
Having said that, I had another look at this and found they might be used if a bisq dev activates a certain feature of bitcoinj that is not being used today. So, I will add PeerGroup.setIgnoreHttpSeeds(true) again.